invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js

```diff
@@ -273,6 +263,39 @@ export class RDMDepositForm extends Component {
     header: { className: "inverted brand", style: { cursor: "pointer" } },
   };
 
+  /* TODO FIX ME and inject me */
+  PIDsProviders = [
+    {
+      scheme: "doi",
+      pidProvider: "datacite",
+      pidClient: "zenodo",
+      pidLabel: "DOI",
+      pidPlaceholder: "10.5321/zenodo.123456",
+      canBeManaged: true,
+      canBeUnmanaged: true,
+      btnLabelGetPID: "Get DOI now!",
+    },
+    {
+      scheme: "rn",
```

Comment:
comment: This is a case of what I mentioned in my PR, this will not pass the validation phase because idutils does not have it.

Reply:
Is this still the case?

Reply:
A PR from @Pineirin will fix this in Marshmallow-Utils, and I will update Invenio-RDM-Records accordingly. So once that release chain goes in, this **will not** be a problem 🚀